### ISSUE TITLE
Serum amyloid P (SAP) scintigraphy abnormal

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10536]
**Preferred term label:**
Serum amyloid P (SAP) scintigraphy abnormal

**Synonyms**
SAP SPECT; Iodine 123; 123-Iodine labeled serum amyloid P scintigraphy; amyloid scan

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Amyloidosis, Amyloid A (AA)
Amyloidosis, light-chain (AL)
Familial Mediterranean Fever
Amyloid neuropathy, TTR-related

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Just saying "abnormal result" does not add much more information than [Amyloidosis HP:0011034](https://hpo.jax.org/browse/term/HP:0011034). The HPO does not differentiate findings based on mode of ascertaining (e.g. there are no separate terms for MRI and CT for brain anomalies).
I am wondering if it would be better to add comments to some of the terms and also add a Medical Action Ontology term for this modality.

**Comment by MickeySegal** [comment]
https://hpo.jax.org/browse/term/HP:0011034 specifies testing tissues.  A serum test seems different enough to merit a different finding, just as a macroscopic and a microscopic term are seen as very different.
https://hpo.jax.org/browse/term/HP:0003712
https://hpo.jax.org/browse/term/HP:0100293

**Comment by pnrobinson** [comment]
@MickeySegal   -- makes sense!
Adding new term: 
- ID: HP:6001100
- label: Positive serum amyloid P scintigraphy
- parents: [Amyloidosis (HP:0011034)](https://hpo.jax.org/app/browse/term/HP:0011034)
- pmid: PMID:16564782
- def: Detection of abnormal amyloid deposits in the body by 123I-labeled serum amyloid P component (SAP) scintigraphy.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10536
- synonyms: Positive SAP scan(NONE)
